docs/5-build-experiment.rst

```diff
@@ -12,11 +12,11 @@ The next step is setting up the Ramble workspace and building the code::
 Ramble will build the source code and set up the following workspace directory structure::
 
     workspace_root/
+        ramble/
+        spack/
```

Comment:
(no change needed) Tnis change by itself looks good, although I see from this review we copy the comment showing the tree in at least 4 places. There should probably only be one (and the rest should refer to that function or doc).

Reply:
Agreed.  I added workspace_directory_structure.rst with the directory structure and tried to include it from 5-build-experiment.rst but the included text is not showing up in preview - @alecbcs how to check if once the docs are built, this will work as intended?